configs/rover_configs.yaml

```diff
@@ -130,4 +130,12 @@ profiling:
         # The count of parallel protocol analyzer
         parallels: ${ROVER_PROFILING_TASK_NETWORK_PROTOCOL_ANALYZE_PARALLELS:2}
         # The size of per paralleled analyzer queue
-        queue_size: ${ROVER_PROFILING_TASK_NETWORK_PROTOCOL_ANALYZE_QUEUE_SIZE:5000}
\ No newline at end of file
+        queue_size: ${ROVER_PROFILING_TASK_NETWORK_PROTOCOL_ANALYZE_QUEUE_SIZE:5000}
+        # The profiling config of the protocols
+        sampling:
+          # The HTTP/1.x and HTTP/2.x profiling config
+          http:
+            # The default body encoding when sampling the request
+            default_request_encoding: ${ROVER_PROFILING_TASK_NETWORK_PROTOCOL_ANALYZE_SAMPLING_HTTP_DEFAULT_REQUEST_ENCODING:UTF-8}
+            # The default body encoding when sampling the response
+            default_response_encoding: ${ROVER_PROFILING_TASK_NETWORK_PROTOCOL_ANALYZE_SAMPLING_HTTP_DEFAULT_RESPONSE_ENCODING:UTF-8}
```

Comment:
What happens if not set?

Reply:
For now, I'm not handling this situation, should I add a default value?

Reply:
If the rover exits, rather than running unexpected, it is fine.

Reply:
Sure, I could add a check when the rover start.

Reply:
You could fall back to utf8 or check and fail the booting. Your call.